### PR TITLE
Cloudfoundry port

### PR DESCRIPTION
**Pull request by sarneaud**
This is mostly to make somthing that builds and passes tests.  Fixes
include

* Code that needed updating to use stuff from utils repo
* Code that didn't use URL_PREFIX and tests that broke after it was
  changed to /marketplace
* Code and tests that were dependent on UK email addresses
* Code and tests that broke after base libraries were updated

A few tests that aren't relevant to Australia were removed.

### CODE REVIEW

File: app/main/views/suppliers.py [hunk 1]
```diff
@@ -114,17 +115,21 @@ def update_supplier():
...
-    if not (supplier_form.validate_on_submit() and
-            contact_form.validate_on_submit()):
-        return edit_supplier(supplier_form=supplier_form,
-                             contact_form=contact_form)
+    if not (supplier_form.validate() and contact_form.validate()):
+        response = make_response(edit_supplier(supplier_form=supplier_form, contact_form=contact_form))
+        response.status_code = 400
+        return response
+
+    del supplier_form.csrf_token
```
Comment (matt-sm) [inline app/main/views/suppliers.py]: This is confusing.  Why do we need to del?

File: config.py [hunk 2]
```diff
@@ -33,32 +32,34 @@ class Config(object):
     DM_SUBMISSIONS_BUCKET = None
     DM_ASSETS_URL = None
 
+    DM_HTTP_PROTO = 'http'
+
     DEBUG = False
 
     RESET_PASSWORD_EMAIL_NAME = 'Digital Marketplace Admin'
-    RESET_PASSWORD_EMAIL_FROM = 'enquiries@digitalmarketplace.service.gov.uk'
+    RESET_PASSWORD_EMAIL_FROM = 'marketplace+password@digital.gov.au'
```
Comment (matt-sm) [inline config.py]: these are the same as buyer frontend?

File: app/main/views/suppliers.py (hunk 1 above)
Comment (sarneaud) [inline app/main/views/suppliers.py]: I added a comment that I hope explains things.

File: config.py (hunk 2 above)
Comment (sarneaud) [inline config.py]: The buyer frontend is just using a works-for-now value, but I changed this config to match it until we have something better.

**Pull request merged**